Web/starter_project_barbies/components/Home-page/ImpactStories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Image from 'next/image'
+import impactStroyOne from "../../public/img/home-page/impact-stories/impact-story-1.png"
+import impactStroyTwo from "../../public/img/home-page/impact-stories/impact-story-2.png"
+import impactStroyThree from "../../public/img/home-page/impact-stories/impact-story-3.png"
+import impactStroyFour from "../../public/img/home-page/impact-stories/impact-story-4.png"
+import impactStroyFive from "../../public/img/home-page/impact-stories/impact-story-5.png"
+import impactStroySix from "../../public/img/home-page/impact-stories/impact-story-6.png"
+import impactStroySeven from "../../public/img/home-page/impact-stories/impact-story-7.png"
```

Comment:
curious question, is there a way to import multiple files from one folder in one import declaration?
i am thinking something like `import { impactStroyOne, impactStroyTwo, ...} from '@/public/img/home-page/impact-stories'`
if it is possible, i would highly recommend checking it out in your case